src/pranitaPJun20/BankingOperationScanner.java

```diff
@@ -0,0 +1,219 @@
+						else if(amount<=balanceHdfc) {
+							hdfcDebitCount++;
+							staticDebitCount++;
+							//balanceIcici=balanceIcici-amount;
+							balanceHdfc-=amount;
+						}
+						else 
+							System.out.println("Invalid Transaction");
+						break;	
+						
+					default :
+						System.out.println("Invalid Operation!");
+			}
+		}
+			
+		break;
+			
+	//Case # 3
+			case "BOB" :{
+				switch(operation) {  
+					case "Credit":
+						bobCreditCount++;
+						staticCreditCount=staticCreditCount+1;
+						balanceBob= balanceBob+amount;
+						break;
+						
+					case "Debit":
+						if(amount>balanceBob) 
+							System.out.println("Account has insufficient balance");	
+						else if(amount<=balanceBob) {
+							bobDebitCount++;
+							staticDebitCount++;
+							balanceBob-=amount;
+						}
+						else 
+							System.out.println("Invalid Transaction");
+						break;	
+						
+					default :
+						System.out.println("Invalid Operation!");
+			}
+		}
+			
+		break;
+		
+		//Case # 4
+			case "SBI" :{
+				switch(operation) {   
+					case "Credit":
+						sbiCreditCount++;
+						staticCreditCount=staticCreditCount+1;
+						balanceSbi= balanceSbi+amount;
+						break;
+						
+					case "Debit":
+						if(amount>balanceSbi) 
+							System.out.println("Account has insufficient balance");	
+						else if(amount<=balanceSbi) {
+							sbiDebitCount++;
+							staticDebitCount++;
+							balanceSbi-=amount;
+						}
+						else 
+							System.out.println("Invalid Transaction");
+						break;	
+						
+					default :
+						System.out.println("Invalid Operation!");
+			}
+		}
+			
+		break;
+		
+		//default
+	default :
+		System.out.println("Invalid Bank Name Entered!");
+		}
+	}
+		
+	void displayTotalBalance() {
+		double sum= balanceIcici+balanceSbi+balanceHdfc+balanceBob ;
+		System.out.println("\nCombined bank balance for banks ICICI,HDFC,BOB,SBI is:"+sum);
+	}
+	
+	void displayTotalCreditDebitCount() {
+		System.out.println("\nOverall Debit Operations count is :" +staticDebitCount);
+		System.out.println("\nOverall Credit Operations count is :" +staticCreditCount);
+	}
+	
+	void displayIndividualCreditCount() {
+		System.out.println("\nBank-wise Credit Operation Count:-\nICICI : "+iciciCreditCount+"\nHDFC Bank : "+hdfcCreditCount+"\nBOB : "+bobCreditCount+"\nSBI : "+sbiCreditCount);
+	}
+	
+	void displayIndividualDebitCount() {
+		System.out.println("\nBank-wise Debit Operation Count:-\nICICI Bank : "+iciciDebitCount+"\nHDFC Bank : "+hdfcDebitCount+"\nBOB : "+bobDebitCount+"\nSBI : "+sbiDebitCount);
+	}
+	
+	
+	String displayBankresponse() {
+		if(staticCreditCount>staticDebitCount) {
+			return "*I appreciate your money management skill*";
+		}
+		else {
+			return "*Please spend money wisely*";
+		}
+	}
+	
+	public static void main(String[] args) {
+		BankingOperationScanner bankingOperationScanner = new BankingOperationScanner();
+		Scanner scanner = new Scanner(System.in);
+		for(int i=1;i<=2;i++) {
```

Comment:
Any specific reason of writing for a loop.

Reply:
>Actually 4 credit debit operations were there so i thought to use for loop.But i will update the code with do while loop.Will write it properly.